CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+## Setup for Code Contributions
+
+### Prerequisites
+
+##### MacOS
+- XCode Command Line Tools `xcode-select --install`
+- If you already have the full Xcode installed, go to menu `Xcode -> Open Developer Tool -> More Developer Tools....`
+- Install Node.js >= 16
+
+##### Windows
+- Install current version of Python3
+- Install Visual Studio Code and add the Microsoft C/C++ extension
+- Install Node.js >= 16
+
+##### Linux
+
+- Install Python > 3.7
+- Install `make`, `gcc`, and `g++`
+- Install Node.js >= 16
+
+#### Repository Setup
+- Fork the project.
+- Open a Terminal and navigate to where you want the project repository to exist
+- Clone your forked repository
+    ```sh
+    git clone https://github.com/<your-github-user>/club_website.git
+    ```
+- Navigate into the root directory
+    ```sh
+    cd club_website
+    ```
+- Install required packages
+    ```sh
+    npm install 
+    ```
+- Switch to beta branch
+    ```sh
+    git checkout beta
+    ```
```

Comment:
Flip these two steps. They'll encounter errors if someone adds a package in beta but not main.